2021-12-02 at 17:14:32 | INFO | Neptune Start Up ....
2021-12-02 at 17:14:32 | INFO | API prefix is: [ /api/v1 ]
2021-12-02 at 17:14:33 | INFO | Add Custom query ...
2021-12-02 at 17:14:33 | INFO | Neptune Starting ....
2021-12-02 at 17:14:49 | INFO | Neptune Shutting Down ....
2021-12-02 at 17:14:58 | INFO | Neptune Start Up ....
2021-12-02 at 17:14:58 | INFO | API prefix is: [ /api/v1 ]
2021-12-02 at 17:14:58 | INFO | Neptune Start Up ....
2021-12-02 at 17:14:58 | INFO | API prefix is: [ /api/v1 ]
2021-12-02 at 17:14:58 | INFO | Neptune Start Up ....
2021-12-02 at 17:14:58 | INFO | API prefix is: [ /api/v1 ]
2021-12-02 at 17:14:58 | INFO | Neptune Start Up ....
2021-12-02 at 17:14:58 | INFO | API prefix is: [ /api/v1 ]
2021-12-02 at 17:15:02 | INFO | Add Custom query ...
2021-12-02 at 17:15:02 | INFO | Neptune Starting ....
2021-12-02 at 17:15:02 | INFO | Add Custom query ...
2021-12-02 at 17:15:02 | INFO | Add Custom query ...
2021-12-02 at 17:15:02 | INFO | Neptune Starting ....
2021-12-02 at 17:15:02 | INFO | Neptune Starting ....
2021-12-02 at 17:15:02 | INFO | Add Custom query ...
2021-12-02 at 17:15:02 | INFO | Neptune Starting ....
2021-12-02 at 17:18:10 | ERROR | Exception at tablemodel insert(): (pymysql.err.DataError) (1406, "Data too long for column 's' at row 1")
[SQL: INSERT INTO mysqlsrc.tokens (s) VALUES (%(s)s)]
[parameters: {'s': '123123123硕大的'}]
(Background on this error at: https://sqlalche.me/e/14/9h9h) 
2021-12-02 at 17:18:21 | ERROR | Exception at tablemodel insert(): (pymysql.err.DataError) (1406, "Data too long for column 's' at row 1")
[SQL: INSERT INTO mysqlsrc.tokens (s) VALUES (%(s)s)]
[parameters: {'s': '1231231'}]
(Background on this error at: https://sqlalche.me/e/14/9h9h) 
2021-12-02 at 17:18:27 | ERROR | Exception at tablemodel insert(): tuple index out of range 
2021-12-02 at 17:18:34 | ERROR | Exception at tablemodel insert(): tuple index out of range 
2021-12-02 at 17:18:37 | ERROR | Exception at tablemodel insert(): tuple index out of range 
